easypost/constant.py

```diff
@@ -1,6 +1,10 @@
-SUPPORT_EMAIL = "support@easypost.com"
-TIMEOUT = 60
+# flake8: noqa
+VERSION = "7.13.0"
+VERSION_INFO = [str(number) for number in VERSION.split(".")]
 API_BASE = "https://api.easypost.com/v2"
+TIMEOUT = 60
+
+SUPPORT_EMAIL = "support@easypost.com"
```

Comment:
Nit: Sort these constants

Need a timeout test for the client, or is this happening in the next PR?

Reply:
It's still on my TODO list, will address before we release.

As for the constants, version is first because it will be the one changing most often so wanted easy access to it at the top